styles/tokens/footer.json

```diff
@@ -66,14 +66,34 @@
         "file": "settings",
         "themable": true
       },
-      "hover-color": {
-        "comment": "Toggle trigger hover color on the Footer.",
-        "value": "{purple-light.value}",
-        "attributes": {
-          "category": "color"
+      "hover": {
+        "color": {
+          "comment": "Toggle trigger hover color on the Footer.",
+          "value": "{white.value}",
+          "attributes": {
+            "category": "color"
+          },
+          "file": "settings",
+          "themable": true
         },
-        "file": "settings",
-        "themable": true
+        "font-weight": {
+          "comment": "Toggle trigger hover font-weight on the Footer.",
+          "value": "500",
+          "attributes": {
+            "category": "content"
```

Comment:
This is a number and not a string of content. Could use size here.

Reply:
`500` is a number but there are other values for `font-weight` that are not numbers. Does that mean they should all be `content` or should the numeric ones be switched to `size`? If I change them all to `size` I get compile errors.

Reply:
It matters what we have the value as because when we build the distributed token files with these values. So, if we have the font weight value as "bold" that's a string and should be set as content. If we have the value as 500 like we do here, it should be size since it's a number.

Reply:
got it. Thanks!

Reply:
awesome, good work!